pkg/console/operator/sync_v400.go

```diff
@@ -302,12 +302,27 @@ func (co *consoleOperator) SyncConfigMap(
 		useDefaultCAFile = true
 	}
 
+	var inactivityTimeoutSeconds int = 0
+	oauthClient, oacErr := co.oauthClient.OAuthClients().Get(co.ctx, oauthsub.Stub().Name, metav1.GetOptions{})
+	if oacErr != nil {
+		return nil, false, "FailedGetOAuthClient", oacErr
+	}
+	if oauthClient.AccessTokenInactivityTimeoutSeconds != nil {
+		inactivityTimeoutSeconds = int(*oauthClient.AccessTokenInactivityTimeoutSeconds)
+	} else {
+		oauthConfig, oacErr := co.oauthConfigClient.Get(co.ctx, api.ConfigResourceName, metav1.GetOptions{})
+		if oacErr != nil {
+			return nil, false, "FailedGetOAuthConfigClient", oacErr
+		}
+		inactivityTimeoutSeconds = int(oauthConfig.Spec.TokenConfig.AccessTokenInactivityTimeout.Seconds())
```

Comment:
Can `AccessTokenInactivityTimeout` be `nil`?

Reply:
So based on the [vendor bump](https://github.com/openshift/console-operator/blob/master/vendor/github.com/openshift/api/config/v1/types_oauth.go#L64) I did its not... but the [changed](https://github.com/openshift/api/commit/fa2d21af7625c58fb6bb5c440c2755f940aba217) it recently in upstream.
I guess we shall bump again then :(